datafusion/sqllogictest/test_files/array.slt

```diff
@@ -1577,9 +1577,9 @@ select
 query ????
 select
   array_append(arrow_cast(make_array(), 'LargeList(Null)'), 4),
-  array_append(make_array(), null),
```

Comment:
tests without arrow-case looks valid for me, should we delete them?

Reply:
what is arrow-case?

Reply:
Maybe arrow-cast?

Reply:
right, arrow-cast, no worries, I found we already have such cases in `array.slt`